test/db/cmd/cmd_tcc

```diff
@@ -25,3 +25,49 @@ tfc rax ms (rcx, rdx, r8, r9, stack);
 tfc rax swift (rdi, rsi, rdx, rcx, r8, r9, xmm0, xmm1, xmm2, xmm3, xmm4, r13, r12);
 EOF
 RUN
+
+NAME=tcc
+FILE=-
+CMDS=<<EOF
+e asm.arch=x86
+e asm.bits=32
+?e 1
+tcc eax jeje(ebx, ecx)
+tcc jeje
+tcc-jeje
```

Comment:
```suggestion
tcc- jeje
```
otherwise the newshell won't be able to execute this properly.
newshell parses the line and it splits it into `<cmd-name>` and `<arguments>`. Since we have all kinds of commands and `-` is not in all cases the last character of the command name, it is required to have a space to separate the `<cmd-name>` from `<arguments>`.

Reply:
This should fix the newshell tests.